### ISSUE TITLE
Improve accessibility for screen readers

### ISSUE THREAD

**Issue by jkupczak** [antonreshetov/mysigmail#6]
Signatures should be improved by adding some code for accessibility.

1. All `<table>`s should have the `role="presentation"` attribute on them. Without this, screen readers will read out the the table rows and columns.
2. All images need an `alt=""` attribute.
3. Adding a `lang=""` attribute with the correct language to the outer most element would be a good idea too. You'd need to ask the user to pick this obviously.

All of this is covered in detail here: https://emails.hteumeuleu.com/email-accessibility-in-action-f7f9d945cf67

**Comment by antonreshetov** [comment]
Closed because it was fixed in #29
